src/components/messenger/chat/styles.scss

```diff
@@ -190,9 +190,11 @@ $recent-indicator-size: 8px;
     text-overflow: ellipsis;
     display: inline-block;
     font-weight: 400;
-    font-size: 12px;
-    line-height: 15px;
-    color: theme.$color-greyscale-11;
+    font-size: 10px;
+    line-height: normal;
+    font-family: 'Roboto Mono';
```

Comment:
Hey Ratik, this probably doesn't set the font-family to 'Roboto Mono' due to the underlying font settings. We'll need to update so we can use fonts like `Roboto Mono`